packages/Laravel/tests/Rector/StaticCall/RequestStaticValidateToInjectRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Laravel\Tests\Rector\StaticCall\RequestStaticValidateToInjectRector\Fixture;
+
+use Illuminate\Http\Request;
+
+class SomeClass
+{
+    public function store()
+    {
+        $validatedData = Request::validate(['some_attribute' => 'required']);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Laravel\Tests\Rector\StaticCall\RequestStaticValidateToInjectRector\Fixture;
+
+use Illuminate\Http\Request;
+
+class SomeClass
+{
+    public function store(\Illuminate\Http\Request $request)
```

Comment:
You are `use`-ing the class above already, no need to fully specify it here.

Reply:
This is job of coding standard. Rector is doing the hard work, ECS makes it nice again :)

See https://github.com/rectorphp/rector#how-to-apply-coding-standards

Reply:
Ah yeah, alright. Everything's fine then I guess 👍